### PR TITLE
build: Rename master to main

### PR DESCRIPTION
**Pull request by gbouv**

### CODE REVIEW

File: dist/config.yaml [hunk 1]
```diff
@@ -124,6 +124,9 @@ gomod:
 announce:
   twitter:
     message_template: '{{ .ProjectName }} {{ .Tag }} is out! Check it out at {{ .ReleaseURL }}'
+  mastodon:
```
Comment (h4ck3rk3y) [inline dist/config.yaml:127]: where does this come from?

File: dist/metadata.json [hunk 2]
```diff
@@ -1 +1 @@
-{"project_name":"kudet-release-artifacts","tag":"0.1.4","previous_tag":"0.1.3","version":"e9b06f0","commit":"e9b06f0ee3f1a79fc43714be40dead4029839f76","date":"2022-07-05T17:28:30.746397-03:00","runtime":{"goos":"darwin","goarch":"arm64"}}
\ No newline at end of file
+{"project_name":"kudet-release-artifacts","tag":"0.1.10","previous_tag":"v0.1.9","version":"1e780f7","commit":"1e780f72e939b2efeff28521cc3e9ca52b38f0a7","date":"2023-02-20T12:03:05.489762+01:00","runtime":{"goos":"darwin","goarch":"arm64"}}
```
Comment (h4ck3rk3y) [inline dist/metadata.json:1]: why is dist checked in?

**Review by h4ck3rk3y — COMMENTED** [review]
Why are the binaries checked in?

File: dist/config.yaml (hunk 1 above)
Comment (gbouv) [reply dist/config.yaml:127]: I ... have no idea TBH. I just ran ./script/build.sh... I can remove them for now, this is not the purpose of this PR anyway. Maybe @tedim52 knows?

File: dist/metadata.json (hunk 2 above)
Comment (gbouv) [reply dist/metadata.json:1]: I also have no idea, pulling @tedim52 here. I was thinking maybe it's how we pull the compiled bin directly in our various circle job? But that seems weird, and I agree we should not use docker as a binary store IMO. If we can remove them, let's remove them

Comment (gbouv) [reply dist/metadata.json]: Tedi is investigating why dist are checked in here, I'd expect it to be removed soon. https://kurtosistech.slack.com/archives/C032ZB39AHH/p1676977397934609
In the meantime I removed it from this PR

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**